lib/components/button/button.dart

```diff
@@ -0,0 +1,41 @@
+import 'package:flutter/material.dart';
+import 'package:flutter/widgets.dart';
+import 'package:flutter_platform_widgets/flutter_platform_widgets.dart';
+
+class Button extends StatelessWidget {
+  final String? title;
+  final VoidCallback? onPressed;
+  final bool isEnabled;
+  const Button({
+    Key? key,
+    this.title,
+    this.onPressed,
+    this.isEnabled = true,
+  }) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+      decoration: BoxDecoration(
+        color: isEnabled ? Colors.white : Colors.grey,
+        borderRadius: BorderRadius.circular(12.0),
```

Comment:
since you're rounding its borders, should we call this class something more specific like `RoundedCornerButton`? 😄  later you might have another maybe `CircularButton` etc..

Reply:
@sleepylee I just try to create a base UI element for this app, so in this app button always begin with white background and rounded corner (until now) lol
Maybe add a button type: primary, normal.. is make sense, but I think we can keep it simple in this project 😁

Reply:
I think later you will have another floating button (circle) from the design no? 🤔

Reply:
Yea, I update this button in another [PR](https://github.com/markgravity/flutter-survey/blob/fac8ce972635d296c0dde43045ebdf392347ce1a/lib/components/button/button.dart#L11) 😁 
I added an option to change this border radius 😓